audit-base/src/main/java/com/wultra/core/audit/base/database/DatabaseAudit.java

```diff
@@ -247,7 +249,16 @@ private void logInternal(String message, AuditLevel level, AuditDetail detail, O
             return;
         }
         final AuditRecord auditRecord = new AuditRecord(message, level, detail.getType(), detail.getParam(), args);
-        writer.write(auditRecord);
+        write(auditRecord);
+    }
+
+    private void write(AuditRecord auditRecord) {
+        try {
+            writer.write(auditRecord);
+        } catch (Exception ex) {
+            logger.debug(ex.getMessage(), ex);
```

Comment:
Is there any convention about log ordering? I would expect to log more serious levels earlier.
I guess that we may write message even for the debug level.
```suggestion
            logger.debug("Audit failed, error: {}", ex.getMessage(), ex);
```

Reply:
I am using the convention we usually use.